### ISSUE TITLE
Items on map cannot be opened in new tab

### ISSUE THREAD

**Issue by brunobesson** [c2corg/v6_ui#1159]
Currently:
* right-clicking doesn't offer to open waypoint/outing/... in new tab (it's not a link)
* midlle-clicking doesn't work
* ctrl-clicking opens the waypoint/outing in current tab

It would be nice to allow these commons behaviours on desktop.

**Comment by ginold** [comment]
right clicking, middle clicking and ctrl clicking are all working only if there is a link (a href).

**Comment by brunobesson** [comment]
>right clicking, middle clicking and ctrl clicking are all working only if there is a link (a href).
Yeah, I know, thanks. But you're not answering the point :wink: It is - to me - a usability problem which we should try to solve one way or another. There are several technical possibilities.
